code/modules/jobs/job_types/cargo_service.dm

```diff
@@ -267,8 +267,8 @@ Janitor
 
 	outfit = /datum/outfit/job/janitor
 
-	access = list(access_janitor, access_maint_tunnels)
-	minimal_access = list(access_janitor, access_maint_tunnels)
+	access = list(access_janitor)
+	minimal_access = list(access_janitor)
```

Comment:
Mouse traps kind of make sense, but the point of maintenance in real life is the authorized personnel use it to maintain wiring/machinery etc.